templates/code-of-conduct.html

```diff
@@ -3,41 +3,60 @@
    rules to make it abundantly clear that this kind of behaviour is not acceptable, so
-   that everyone can feel comfortable attending EMF.
+   that everyone can feel comfortable attending EMF. Everybody in the EMF community is required
+   to abide by this code of conduct, both at the event and on any online channels.
 </p>
 
-<p>Everyone who visits EMF is required to abide by this code of conduct, both at the event
-   and on any online channels. We will not tolerate harassment of participants or
-   discriminatory behaviour of any form.
-   Specifically:</p>
+<h3>The Rules</h3>
+
+<p>We do not tolerate harassment or discriminatory behaviour of any form. Specifically:</p>
 
 <ul>
   <li>Do not engage in homophobic, racist, transphobic, ableist, sexist, or otherwise
       prejudiced behaviour.</li>
-  <li>Do not harass people. Stalking, unconsented physical contact, or sexual attention is harassment.
-      Dressing or acting in a certain way is not consent.</li>
-  <li>Every participant's personal space is their own. If you are asked to leave someone alone, or to
+  <li>Do not harass people. Stalking, unconsented physical contact, threatening behaviour, and unwanted 
+      sexual attention is harassment. Dressing or acting in a certain way is not consent.</li>
+  <li>Everyone's personal space is their own. If you are asked to leave someone alone, or to
       leave their village, you must respect this.</li>
-  <li>Some attendees may not want to be filmed or photographed. Respect their wishes.</li>
+  <li>Some people may not want to be filmed or photographed. You must have consent from anyone
+      who is identifiable in a photo.</li>
   <li>Aggression and elitism are not welcome &mdash; nobody should be afraid to ask questions.</li>
 </ul>
 
-<p>If you break these rules, we may expel you from the event without a refund,
-   and we may ban you from future EMF events.</p>
+<h3>Reporting an Incident</h3>
+
 <p>If you are being harassed or witness a breach of this code of conduct,
   you can get in touch with us in the following ways:</p>
 <ul>
   <li>Find a member of event staff (they'll be clearly identifiable) and let them know.
-  All our volunteer staff will be briefed on how to handle a harassment situation.</li>
+  All our volunteer staff will be briefed on how to handle reports.</li>
   <li>Drop us an email at <a href="mailto:{{ config['DUTY_EMAIL'][1] }}">{{ config['DUTY_EMAIL'][1] }}</a>.</li>
 </ul>
-<p>If you have any comments or queries, you can get in touch with our Conduct &amp; Accessibility team at <a href="mailto:{{ config['CONDUCT_EMAIL'][1] }}">{{ config['CONDUCT_EMAIL'][1] }}</a>.</p>
 
-<p><small>This Code of Conduct is licensed under the <a href="https://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 License</a>.</small></p>
+<h3>Our Response</h3>
+
+<p>During our events, we aim to deal with all conduct issues within 12 hours. 
```

Comment:
Is it worth specifying the response time when we're not running an event? (Apols for slow reply)

Reply:
Ideally yes, but I don't want to make promises we can't keep, so I have no idea what timeframe would be reasonable.

Reply:
Is "deal with" implying that we'll finish handling with the issue within that 12 hour period, or just that our initial response will be within 12 hours?

Reply:
Good point, that's not terribly clear. I changed it to "reach a decision".

Reply:
In which case we should say something like "outside of events, we aim to initially respond within one week".

I don't feel comfortable committing to a resolution time outside events, but I feel we can at least say we'll respond saying "we have received your email" within a week or so.

Reply:
I think that's fair, I'll add that. I'd like to be able to promise a shorter time than that but we need to make sure we'll meet it.